h2o-py/docs/intro.rst

```diff
@@ -42,7 +42,7 @@ produce a solution faster. The conceptual paradigm MapReduce (AKA "divide and co
 and combine"), along with a good concurrent application structure,
 (c.f. jsr166y and NonBlockingHashMap) enable this type of scaling in H2O.
 
-For application developers and data scientists, the gritty details of thread-safety, algorithm parallelism, and node coherence on a network are concealed by simple-to-use REST calls that are all documented here. In addition, H2O is an open-source project under the Apache v2 licence. All of the source code is on `github <https://github.com/h2oai/h2o-3>`_. 
+For application developers and data scientists, the gritty details of thread-safety, algorithm parallelism, and node coherence on a network are concealed by simple-to-use REST calls that are all documented here. In addition, H2O is an open-source project under the GNU Affero General Public License v3.0 licence. All of the source code is on `github <https://github.com/h2oai/h2o-3>`_. 
 
```

Comment:
Here, perhaps

GNU Affero General Public License v3.0

should be GNU Affero General Public License v3

Since Apache v2 and not Apache v2.0